automation/taskcluster/l10n/locales.py

```diff
@@ -11,11 +11,23 @@
 OPEN_LOCALES = "locales = ["
 CLOSE_LOCALES = "]"
 
+# Android uses non-standard locale codes, these are the mappings back and forth
+ANDROID_LEGACY_MAP = {
```

Comment:
Do we know if this is all of them? (or just going through the list of missing ones Delphine mentioned)

Reply:
If there's android documentation for these non-standard mappings, include a link to it in the comment - consider what someone might need to look through, once we start adding more locales (that could also be non-standard).

Reply:
It seems like its only these three. I'll add a link to the docs